README_comments.md

```diff
@@ -0,0 +1,247 @@
+administrator) to do that. This will almost certainly be required if you want
+to run the workflow on a high-performance computing (HPC) cluster. 
+
+> **NOTE:**
+> The workflow has been tested with the following Singularity versions:  
+>  * `v2.6.2`
+>  * `v3.5.2`
+
+After installing Singularity, install the remaining dependencies with:
+```bash
+conda env create -f install/environment.yml
+```
+
+
+### As root user on Linux
+
+If you have a Linux machine, as well as root privileges, (e.g., if you plan to
+run the workflow on your own computer), you can execute the following command
+to include Singularity in the Conda environment:
+
+```bash
+conda env create -f install/environment.root.yml
+```
+
+## 4. Activate environment
+
+Activate the Conda environment with:
+
+```bash
+conda activate zarp
+```
+
+# Extra installation steps (optional)
+
+## 5. Non-essential dependencies installation
+
+Most tests have additional dependencies. If you are planning to run tests, you
+will need to install these by executing the following command _in your active
+Conda environment_:
+
+```bash
+conda env update -f install/environment.dev.yml
+```
+
+## 6. Successful installation tests
+
+We have prepared several tests to check the integrity of the workflow and its
+components. These can be found in subdirectories of the `tests/` directory. 
+The most critical of these tests enable you to execute the entire workflow on a 
+set of small example input files. Note that for this and other tests to complete
+successfully, [additional dependencies](#installing-non-essential-dependencies) 
+need to be installed. 
+Execute one of the following commands to run the test workflow 
+on your local machine:
+* Test workflow on local machine with **Singularity**:
+```bash
+bash tests/test_integration_workflow/test.local.sh
+```
+* Test workflow on local machine with **Conda**:
+```bash
+bash tests/test_integration_workflow_with_conda/test.local.sh
+```
+Execute one of the following commands to run the test workflow 
+on a [Slurm][slurm]-managed high-performance computing (HPC) cluster:
+
+* Test workflow with **Singularity**:
+
+```bash
+bash tests/test_integration_workflow/test.slurm.sh
+```
+* Test workflow with **Conda**:
+
+```bash
+bash tests/test_integration_workflow_with_conda/test.slurm.sh
+```
+
+> **NOTE:** Depending on the configuration of your Slurm installation you may
+> need to adapt file `slurm-config.json` (located directly under `profiles`
+> directory) and the arguments to options `--cores` and `--jobs`
+> in the file `config.yaml` of a respective profile.
+> Consult the manual of your workload manager as well as the section of the
+> Snakemake manual dealing with [profiles].
+
+# Running the workflow on your own samples
+
+1. Assuming that your current directory is the repository's root directory,
+create a directory for your workflow run and move into it with:
+
+    ```bash
+    mkdir config/my_run
```

Comment:
It does not make sense to have the runs under the config directory. Best would be to have a separate directory (something like _runs_) and place individual runs in subdirectories thereof.